src/ui/close-button.svelte

```diff
@@ -1,11 +1,13 @@
 <script lang="ts">
-  import Icon from "../assets/close.svg";
+  import Icon from "../assets/close.svg?raw";
```

Comment:
Just curious, what does this mean?

Reply:
It reads the svs as row data, I need to do this in order to change the fill color of the close button